tests/dragonfly/cluster_test.py

```diff
@@ -1773,6 +1773,53 @@ async def node1size0():
         assert str(i) == await nodes[1].client.get(f"{{key50}}:{i}")
 
 
+@pytest.mark.parametrize("type", ["list", "hash", "string", "set", "zset"])
+@dfly_args({"proactor_threads": 2, "cluster_mode": "yes"})
+@pytest.mark.asyncio
+async def test_cluster_migration_huge_list(df_factory: DflyInstanceFactory, type):
+    instances = [
+        df_factory.create(port=BASE_PORT + i, admin_port=BASE_PORT + i + 1000) for i in range(2)
+    ]
+    df_factory.start_all(instances)
+
+    nodes = [await create_node_info(instance) for instance in instances]
+    nodes[0].slots = [(0, 16383)]
+    nodes[1].slots = []
+
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
+
+    logging.debug(f"Generating huge {type}")
```

Comment:
check out test_big_containers how it uses StaticSeeder
if we can do the same here and use the capture to compare
also this will be one test case and not running for each data type